main.go

```diff
@@ -210,4 +211,11 @@ func initFlags(fs *pflag.FlagSet) {
 		10*time.Minute,
 		"The minimum interval at which watched resources are reconciled.",
 	)
+
+	fs.StringVar(
+		&options.PowerVSProviderIDFormat,
+		"powervs-provider-id-fmt",
+		"v1",
+		"ProviderID format is used set the Provider ID format for Machine",
+	)
```

Comment:
this accepts any random value as of now, can we limit this to accept only v1 or v2